network/src/p2p/handler.rs

```diff
@@ -16,9 +16,10 @@
 
 use std::collections::{BTreeSet, HashMap, VecDeque};
 use std::iter::FromIterator;
+use std::marker::{Send, Sync};
```

Comment:
You don't need these imports.
https://doc.rust-lang.org/std/prelude/index.html

Reply:
Done